locksmith/src/controllers/v2/metadataController.ts

```diff
@@ -246,3 +247,29 @@ export const updateUsersMetadata: RequestHandler = async (
     error,
   })
 }
+
+export const readUserMetadata: RequestHandler = async (request, response) => {
+  const userAddress = Normalizer.ethereumAddress(request.params.userAddress)
+  const tokenAddress = Normalizer.ethereumAddress(request.params.lockAddress)
+  const loggedInUser = request.user!.walletAddress
+  const normalisedLoggedInAddress = Normalizer.ethereumAddress(loggedInUser)
+  if (normalisedLoggedInAddress !== userAddress) {
```

Comment:
I think it's ok to _not_ yield an error as long as as we don't serve the private part? (see below)